source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -304,14 +341,17 @@ namespace
 
         for(size_t i = 0; i < numFrames; i++)
         {
+            printProgress(i + 1, numFrames);
             const auto beforeCapture = HighResClock::now();
             const auto frame = Zivid::HDR::capture(camera, settingsVector);
             const auto afterCapture = HighResClock::now();
             frame.getPointCloud();
             const auto afterProcess = HighResClock::now();
 
             captureDurations.push_back(afterCapture - beforeCapture);
+            addToRawFile(std::string("capture duration: ") + formatDuration(afterCapture - beforeCapture));
             processDurations.push_back(afterProcess - afterCapture);
+            addToRawFile(std::string("process duration: ") + formatDuration(afterProcess - afterCapture));
```

Comment:
Can we add '3D' here to make it consistent with 2D capture?

Reply:
sure :)